contrib/fw-in-container/fw-in-container

```diff
@@ -235,6 +235,7 @@ journalctl -u firewalld.service --since '3 min ago'
 m
 make
 make install
+make check TESTSUITEFLAGS="-j4"
```

Comment:
Could do:
```
make check TESTSUITEFLAGS="-j$(nproc)"
```

Reply:
This line is in the pre-generated bash history of the container, and the point is to type `ARROWUP`, `PAGEUP` or `CTRL+R` to find the useful command. Usually, followed by editing the line found there, e.g. by adding a `-k FILTER` option.

Anyway. I care that there is an example at hand, how to run the testsuite in parallel. Changed.